clients/benchmarks/CMakeLists.txt

```diff
@@ -131,6 +131,8 @@ else( )
   target_link_libraries( hipblas-bench PRIVATE ${CUDA_LIBRARIES} Threads::Threads )
 endif( )
 
+target_compile_definitions( hipblas-bench PRIVATE __BUILD_WITH_SOLVER__ )
+
```

Comment:
underscores often used for special meaning, I would suggest just HIPBLAS_WITH_SOLVER ?

Reply:
@daineAMD HIP by itself would indicate to me the hip layer so that is why I suggested simply "HIPBLAS_WITH_SOLVER"
This is some advice I found in a search "According to the C++ Standard, identifiers starting with one underscore are reserved for libraries. Identifiers starting with two underscores are reserved for compiler vendors."

Reply:
I changed it to use __HIP_PLATFORM_SOLVER__ which was added in https://github.com/ROCmSoftwarePlatform/hipBLAS/commit/f9c8c54da7ef7bce2189f2bbb499138322ee3d4a. I can change it in another PR since it's probably out of the scope of this one.